### ISSUE TITLE
[Discussion] Skip "Create expense" when clicking "mark as paid"?

### ISSUE THREAD

**Issue by Shoggomo** [spliit-app/spliit#220]
When clicking the "mark as paid" button in the Balances tab we get directed to a prefilled "Create expense" form. I assume, that the prefilled data should be correct for most cases (provided #197 is fixed). So I think we could skip this form, create the expense automatically and keep the user in the balance tab, so the result can be seen.

Possible changes to this expense can still be made from the Expenses tab.
What do other people think?
Is the direction to the "Create expense" form, when clicking the "mark as paid" button, useful, or should we skip it?

## Mockup
![grafik](https://github.com/user-attachments/assets/4edbd725-8774-464e-a692-3ef9cb1a2bba)

**Comment by totti-rdz** [comment]
I agree with @CarliJoy. I like to double-check things and this screens allows me to do so. Furthermore I can add a more specific title if I want to or attach a document (maybe PayPal screenshot).

**Comment by maze88** [comment]
I think it's important that there are **at least two actions, to prevent accidental reimbursement** (knowingly and -even worse- _un_knowingly!).

Partially related: I don't think logging reimbursements on the _Expenses_ tab makes sense. Instead I would suggest having another list on the _Balances_ tab, logging reimbursements; with lines something like:

> _Jane [paid pack/tallied up with/settled with] John_ ([amount]).

**Comment by gtkacz** [comment]
I think a simple filter to not show reimbursements on the expenses page would go a long way

**Comment by aChrisYouKnow** [comment]
I had a friend recently who would be settling up for multiple people, and he was wishing there was a multi select to mark multiple people as paid. In that case, the desire would be some simple confirmation that they wanted to settle up for those people, plus the total. If that was of interest to more people, perhaps whatever button did that could serve as this method of skipping the 'create expense' form, while still requiring two clicks.

**Comment by antonio-ivanovski** [comment]
I recently improved this flow in my fork: clicking **Mark as paid** now shows a simple summary modal of what will be created instead of opening the full expense form. There is also the edit button that opens up the full create expense page.

<img width="350" alt="Image" src="https://github.com/user-attachments/assets/2bc6ffb9-cf90-4442-a9fd-ce6b179ff569" />

I also like the idea of tagging reimbursements so they can be filtered in the expense list, and I plan to add that (and more filters) soon.

Repo: https://github.com/antonio-ivanovski/spliit-cloud
Public instance: https://spliit.cloud/

**Comment by antonio-ivanovski** [comment]
> I think a simple filter to not show reimbursements on the expenses page would go a long way

Done ✅ 

<img width="770" height="582" alt="Image" src="https://github.com/user-attachments/assets/4b7b1f67-5e7d-4c4a-81eb-2743e1e037ef" />
